common/src/test/java/io/netty/util/HashedWheelTimerTest.java

```diff
@@ -50,11 +50,12 @@ public void testScheduleTimeoutShouldRunAfterDelay() throws InterruptedException
         final CountDownLatch barrier = new CountDownLatch(1);
         final Timeout timeout = timer.newTimeout(timeout1 -> barrier.countDown(), 2, TimeUnit.SECONDS);
         assertTrue(barrier.await(3, TimeUnit.SECONDS));
-        assertTrue("timer should expire", timeout.isExpired());
+        assertTrue(timeout.isExpired(), "timer should expire");
         timer.stop();
     }
 
-    @Test(timeout = 3000)
+    @Test
+    @org.junit.jupiter.api.Timeout(value = 3000, unit = TimeUnit.MILLISECONDS)
```

Comment:
Why we've FQCN instead of importing it?

Reply:
`Timeout` class exists in the `io.netty.util` package.